packages/common-substrate/src/project/versioned/index.ts

```diff
@@ -3,7 +3,3 @@
 
 export * from './ProjectManifestVersioned';
 export * from './v0_0_1';
-export * from './v0_2_0';
-export * from './v0_2_1';
-export * from './v0_3_0';
-export * from './v1_0_0';
```

Comment:
Can we please only have support for v1.0.0 and all of its features, we don't need to support any earlier versions